staging/src/k8s.io/kubectl/pkg/cmd/taint/taint.go

```diff
@@ -219,7 +219,7 @@ func (o TaintOptions) validateFlags() error {
 // Validate checks to the TaintOptions to see if there is sufficient information run the command.
 func (o TaintOptions) Validate() error {
 	resourceType := strings.ToLower(o.resources[0])
-	validResources, isValidResource := []string{"node", "nodes"}, false
+	validResources, isValidResource := []string{"node", "nodes", "no"}, false
```

Comment:
This is very hacky, I'd prefer you use `o.Mapper.KindFor` instead and then just verify if that's node or not. That's the preferred approach.

Reply:
Sorry, I have some doubts, can you describe the details